### ISSUE TITLE
Bottom padding of recycler view offsets the dragging item [Drag drop]

### ISSUE THREAD

**Issue by vinayak-garg** [h6ah4i/android-advancedrecyclerview#131]
Hi,
I saw an issue which I was able to reproduce easily with example app. 

Steps-
1. Modify the `myRecyclerViewStyle` in v11/styles.xml to something like below. So that there is sufficient padding at bottom of recycler view.  
```
        <item name="android:paddingBottom">90dp</item>
        <item name="android:paddingLeft">0dp</item>
        <item name="android:paddingRight">0dp</item>
        <item name="android:paddingTop">0dp</item>
```
2. Now open the example **Draggable (Grid Layout)**
3. Scroll to bottom, such that the bottom padding is mostly hidden and item **51-Z** is almost aligned to bottom screen.
4. Try to grab item **51-Z**, the shadow item does not appear at the right position. It appears much above.

![grid_layout_bug](https://cloud.githubusercontent.com/assets/1853535/10601066/902edb44-772c-11e5-841c-ee6bd376e3fd.png)

Refer to the screen capture above. Notice how my finger is above original position of item **51-Z**, but the (rotated) shadow item is much above.

Note - I also added 

        android:layout_marginTop="4dp"
        android:layout_marginLeft="4dp"

to the RecyclerView xml item.

**Comment by h6ah4i** [comment]
@vinayak-garg Thanks for the issue report! I have fixed this issue and pushed it to the [fix_issue_131 branch](https://github.com/h6ah4i/android-advancedrecyclerview/tree/fix_issue_131). Could you try it?

**Comment by vinayak-garg** [comment]
@h6ah4i Thanks for fixing this. I tried the branch and it is working great!

**Comment by h6ah4i** [comment]
Fixed in v0.8.3.  

@vinayak-garg Thank you for helping me again! I really appreciate it ;)
